### PR TITLE
TC__03.08.09 | +New Item > Verify "Copy from" field >can't create new item from none existing item

### PR DESCRIPTION
**Pull request by ReshesOxana**
https://trello.com/c/hBoOycfB/271-tc030809-new-item-verify-copy-from-field-cant-create-new-item-from-none-existing-item

### CODE REVIEW

File: cypress/e2e/verifyCopyFromField.cy.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+///<reference types ="cypress"/>
+import { jobName, rendomJob } from "../fixtures/createNewJob.json";
+
+describe('Verify "Copy from" field', () => {
+    beforeEach('Create new job', () => {
+        cy.get("a[href='/view/all/newJob']").click()
+        cy.get('label[for=name]').should('have.text', 'Enter an item name')
```
Comment (Rom-T) [inline cypress/e2e/verifyCopyFromField.cy.js:7]: Please move `Enter an item name` to fixtures

File: cypress/e2e/verifyCopyFromField.cy.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        cy.get('input#name.jenkins-input').type(jobName)
+        cy.get('li[tabindex="0"] span').should('contain', 'Freestyle project').click({ multiple: true })
```
Comment (Rom-T) [inline cypress/e2e/verifyCopyFromField.cy.js:9]: Please move `Freestyle project` to fixtures. 
Why use { multiple: true } here?

File: cypress/e2e/verifyCopyFromField.cy.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+        cy.get('#ok-button').click()
+        cy.get('.jenkins-app-bar__content > h1').should('have.text', 'Configuration')
```
Comment (Rom-T) [inline cypress/e2e/verifyCopyFromField.cy.js:11]: Please move `Configuration` to fixtures

File: cypress/e2e/verifyCopyFromField.cy.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    })
+    it("TC__03.08.09 | New Item > Verify 'Copy from' field >can't create a new item from none existing item", () => {
+        cy.get("#jenkins-name-icon").click()
+        cy.get("a[href='/view/all/newJob']").click()
+        cy.get('#name.jenkins-input').type('j')
+        cy.get('div.item-copy').click()
+        cy.get('#from.jenkins-input.auto-complete').type(rendomJob)
+        cy.get('#ok-button').click()
+
+        cy.get('h1').should('contain', 'Error')
```
Comment (Rom-T) [inline cypress/e2e/verifyCopyFromField.cy.js:22]: Please move `Error` to fixtures

File: cypress/e2e/verifyCopyFromField.cy.js [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+        cy.get('input#name.jenkins-input').type(jobName)
+        cy.get('li[tabindex="0"] span').should('contain', 'Freestyle project').click({ multiple: true })
+        cy.get('#ok-button').click()
+        cy.get('.jenkins-app-bar__content > h1').should('have.text', 'Configuration')
+        cy.get('.jenkins-button--primary ').click()
+    })
+    it("TC__03.08.09 | New Item > Verify 'Copy from' field >can't create a new item from none existing item", () => {
+        cy.get("#jenkins-name-icon").click()
+        cy.get("a[href='/view/all/newJob']").click()
+        cy.get('#name.jenkins-input').type('j')
```
Comment (Rom-T) [inline cypress/e2e/verifyCopyFromField.cy.js:17]: Please move `j` to fixtures

File: cypress/e2e/verifyCopyFromField.cy.js (hunk 2 above)
Comment (ReshesOxana) [reply cypress/e2e/verifyCopyFromField.cy.js:9]: @Rom-T {multiple: true} is allows to interact with multiple DOM elements that meet the criteria.

File: cypress/e2e/verifyCopyFromField.cy.js (hunk 3 above)
Comment (ReshesOxana) [reply cypress/e2e/verifyCopyFromField.cy.js]: @Rom-T please check, all done.

File: cypress/e2e/verifyCopyFromField.cy.js (hunk 2 above)
Comment (Rom-T) [reply cypress/e2e/verifyCopyFromField.cy.js]: @ReshesOxana Why not find a locator specifically for this element?
`cy.get('.hudson_model_FreeStyleProject span')`

File: cypress/e2e/verifyCopyFromField.cy.js (hunk 1 above)
Comment (Rom-T) [reply cypress/e2e/verifyCopyFromField.cy.js:7]: This one was forgotten

**Review by Rom-T — APPROVED** [review]

**Review by S-Lanchik — APPROVED** [review]

**Pull request merged**